### PR TITLE
[14.0][MIG] base_report_auto_create_qweb

### PR DESCRIPTION
**Pull request by SimoRubi**
Standard migration from https://github.com/OCA/server-tools/commit/a36ec5ee8be821c887687a5b3999445b38fff416

### CODE REVIEW

File: base_report_auto_create_qweb/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+When creating a report in Settings > Technical > Actions > Reports or
```
Comment (tafaRU) [inline base_report_auto_create_qweb/readme/DESCRIPTION.rst:1]: this menu doesn't exist anymore in 14.0, please replace it by the following:
`Settings > Technical > Actions > Reports`

**Review by tafaRU — CHANGES_REQUESTED** [review]

File: base_report_auto_create_qweb/readme/DESCRIPTION.rst (hunk 1 above)
Comment (tafaRU) [reply base_report_auto_create_qweb/readme/DESCRIPTION.rst:1]: it'ok, I misread it!

**Review by tafaRU — APPROVED** [review]

**Comment by SimoRubi** [comment]
@OCA/tools-maintainers can this be merged?
Thanks!

**Review by simahawk — APPROVED** [review]
just code review, LG

**Pull request merged**